src/psyclone/tests/psyir/transformations/read_only_verify_trans_test.py

```diff
@@ -105,7 +105,7 @@ def test_read_only_options():
                     options={"region_name": ("a", "b")})
     code = str(psy.gen)
 
-    assert 'CALL read_only_verify_psy_data % PreStart("a", "b", 4, 4)' in code
+    assert 'CALL read_only_verify_psy_data % PreStart("a", "b", 5, 5)' in code
```

Comment:
Why do we only get `j` added here? To some degree that is correct (since `i` is a kind of 'local only' variable, but ... I actually don't know why it is not added.  The expression as kernel child has both `i` and `j`.

Reply:
It is what read_only_verify is provided with as pre_list `src/psyclone/psyir/nodes/value_range_check_node.py::85`
```
read_write_info = ctu.get_in_out_parameters(self)
return {'pre_var_list': read_write_info.read_list,
        'post_var_list': read_write_info.write_list}
```
here read_list means input variables to the selected node.

Note that this is different from exctraction `src/psyclone/psyir/nodes/extract_node.py:203` which provides all used variables in the pre_list:
```
  options = {'pre_var_list': read_write_info.all_used_vars_list,
             'post_var_list': read_write_info.write_list,
             'post_var_postfix': self._post_name}
```

Reply:
Done as suggested